src/__tests__/core/index.test.js

```diff
@@ -47,19 +49,199 @@ describe('setup', () => {
     const model = mock.calls[0][1].toJS();
     expect(model.auth.redirectUrl).toBe('https://test.com/path/');
   });
+
   it('should work with redirect:false and responseType:id_token', () => {
     const options = {
       auth: {
         redirect: false,
         responseType: 'id_token'
       }
     };
+
     setup('id', 'clientID', 'domain', options, 'hookRunner', 'emitEventFn', 'handleEventFn');
     const { mock } = mockInit;
     expect(mock.calls.length).toBe(1);
     const model = mock.calls[0][1].toJS();
     expect(model).toMatchSnapshot();
   });
+
+  describe('clientBaseUrl', () => {
+    it('should default to the specified domain', () => {
+      const { mock } = mockInit;
+
+      setup(
+        'id',
+        'clientID',
+        'my-tenant.us.auth0.com',
+        {},
+        'hookRunner',
+        'emitEventFn',
+        'handleEventFn'
+      );
+
+      expect(mock.calls.length).toBe(1);
+
+      const model = mock.calls[0][1].toJS();
+      expect(model.clientBaseUrl).toBe('https://my-tenant.us.auth0.com');
+    });
+
+    it('should use the clientBaseUrl option if given', () => {
+      const { mock } = mockInit;
+
+      setup(
+        'id',
+        'clientID',
+        'my-tenant.us.auth0.com',
+        {
+          clientBaseUrl: 'https://client-base-url.example.com',
+          configurationBaseUrl: 'https://config-base-url.example.com',
```

Comment:
this key and the above one have the same value. How can I, just by reading this one test, know that the value you end up getting in the end comes from "clientBaseUrl" and not "configurationBaseUrl"?

Reply:
The values are different. Similar, I admit, but different. I can change it so it's more obvious 👍🏻

Reply:
Rereading this, I see now they are different. It's OK to keep them.